tests/testthat/test-lancelot-support.R

```diff
@@ -890,6 +912,30 @@ test_that("lancelot_check_data disallows double fitting to deaths", {
 })
 
 
+test_that("lancelot_check_data disallows double fitting to admissions", {
+  ## Specifically, disallow double fitting to aggregated and
+  ## age-specific hospital admissions
+  data <- lancelot_simple_data(read_csv(sircovid_file("extdata/example.csv")))
+
+  expect_silent(lancelot_check_data(data))
+
+  ## Throw error
+  data$admissions_60_69 <- 1
+  data$all_admission <- 4
+  expect_error(
+    lancelot_check_data(data),
+    "Cannot fit to admissions by age and aggregate together!")
```

Comment:
It would be good to test a version where we fit to age-aggregated and age-specific in the same fit, but not on the same dates

Reply:
sure, will do